app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,70 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :user_name %><br />
+    <%= f.text_field :user_name, autocomplete: "username" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :first_name %><br />
+    <%= f.text_field :first_name, autocomplete: "First Name" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :last_name %><br />
+    <%= f.text_field :last_name, autocomplete: "Last Name" %>
+  </div>
+
+
+  <div class="field">
+    <%= f.label :pass %><br />
+    <%= f.text_field :pass, autocomplete: "Mountain Mass" %>
+  </div>
```

Comment:
Should this be Mountain Pass instead of Mass?

Reply:
Yep. Good catch!